WebRoot/js/zimbraMail/abook/view/ZmContactSplitView.js

```diff
@@ -1207,7 +1208,12 @@ function(contact, params, asHtml, count) {
 	}
 
 	// icon
-	htmlArr[idx++] = AjxImg.getImageHtml(contact.getIcon(folder), null, "id=" + this._getFieldId(contact, "type"),null, null, ["ZmContactIcon"]);
+	var avatarUrl = contact.getImageUrl();
+	if(avatarUrl) {
+		htmlArr[idx++] = "<div class='ZmContactIcon' id='" + this._getFieldId(contact, "type") + "' ><img src='" + avatarUrl + "' alt='" + contact.getFullName() + "' /></div>"; 
```

Comment:
small: We should avoid using string concatenation if possible. Maybe use htmlArr.push ( "string1" , "string2" ) something like that .